docs/source/tm/installation.md

```diff
@@ -0,0 +1,131 @@
+# நிறுவல்
+
+நீங்கள் தொடங்குவதற்கு முன், தகுந்த தொகுப்புகளை நிறுவுவதன் மூலம் உங்கள் சூழலை அமைக்க வேண்டும்.
+
+`huggingface_hub` **Python 3.8+** மின்பொருள்களில் சோதிக்கப்பட்டுள்ளது.
+
+### பிப் மூலம் நிறுவு
+
+**pip மூலம் நிறுவல்**
+
+`huggingface_hub`-ஐ ஒரு [மெய்நிகர் சூழலில்](https://docs.python.org/3/library/venv.html) (virtual environment) நிறுவுவது மிகவும் பரிந்துரைக்கப்படுகிறது. நீங்கள் பைதான்  மெய்நிகர் சூழல்களைக் குறித்து அறியாதவராக இருந்தால், இந்த [வழிகாட்டலைப்](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/)பார்க்கவும். ஒரு மெய்நிகர் சூழல் பல்வேறு திட்டங்களை எளிதில் நிர்வகிக்கவும், சார்புகளுக்கிடையிலான (dependencies) இணக்கமின்மை பிரச்சனைகளைத் தவிர்க்கவும் உதவுகிறது.
+
+முதலில், உங்கள் திட்ட அடைவரிசையில் (project directory) ஒரு மெய்நிகர் சூழலை உருவாக்கத் தொடங்குங்கள்:
+
+```bash
+python -m venv .env
+```
+மெய்நிகர் சூழலை செயல்படுத்தவும். Linux மற்றும் macOS-இல்:
+
+
+```bash
+source .env/bin/activate
+```
+
+Windows-இல் மெய்நிகர் சூழலை செயல்படுத்த:
+
+```bash
+.env/Scripts/activate
+```
+
+இப்போது நீங்கள் `huggingface_hub`-ஐ [PyPi பதிவகத்திலிருந்து](https://pypi.org/project/huggingface-hub/) நிறுவ தயாராக இருக்கிறீர்கள்.
+
+```bash
+pip install --upgrade huggingface_hub
+```
+
+முடித்த பிறகு, [நிறுவல் சரியாக வேலை](#check-installation) செய்கிறதா என்பதைச் சோதிக்கவும்.
+
+### விருப்பத் தேவைப்படும் சார்புகளை நிறுவல்**
+
+`huggingface_hub`-இன் சில சார்புகள் விருப்பமானவை, ஏனெனில் அவை `huggingface_hub`-இன் அடிப்படை அம்சங்களை இயக்க தேவையில்லை. எனினும், விருப்பச் சார்புகள் நிறுவப்படாதால், `huggingface_hub`-இன் சில அம்சங்கள் கிடைக்காது.
+
+நீங்கள் விருப்பத் தேவைப்படும் சார்புகளை `pip` மூலம் நிறுவலாம்:
+
+```bash
+# Install dependencies for tensorflow-specific features
```

Comment:
Translating the code comments would also make it easier for native readers to read :)

Reply:
especially because in this case the comment is also shown to the end reader (see [here](https://huggingface.co/docs/huggingface_hub/installation#install-optional-dependencies))
![image](https://github.com/user-attachments/assets/e15fd083-86b7-42aa-ab29-eb18dcce0b1c)